### PR TITLE
[TRA-286] Add travis and upgrade version

### PR DESCRIPTION
**Pull request by azkalhaq**
Bingung travisnya suruh ngapain. Hehe
cc: @petrabarus

### CODE REVIEW

**Comment by petrabarus** [comment]
How about run lint and checkstyle.

**Comment by petrabarus** [comment]
Run phpcs https://github.com/urbanindo/yii2-queue/blob/master/ruleset.xml

File: composer.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
     ],
     "minimum-stability": "dev",
     "require": {
-        "apache/thrift": "0.9.2",
-        "yiisoft/yii2": "~2.0@dev"
+        "apache/thrift": "0.10.0",
```
Comment (petrabarus) [inline composer.json]: run composer lock

**Comment by petrabarus** [comment]
After this, upgrade the library on the app sample (https://github.com/petrabarus/yii2-thrift-test-app) and then try if the client still can work (https://github.com/petrabarus/yii2-thrift-test-client-php)

**Comment by petrabarus** [comment]
Bonus point: Enable codecov https://codecov.io/gh/urbanindo/yii2-thrift 😆

**Comment by azkalhaq** [comment]
mas @petrabarus ? baru kepikiran...
Kalau upgrade library disini https://github.com/petrabarus/yii2-thrift-test-app
berarti harus buat release dulu ya?
Nah ini release tagnya dibuat di fork-ku atau harus di urbanindo

**Comment by petrabarus** [comment]
Sebenernya gk mesti. Composernya bisa diset pake commit hash kalo buat ngetest.




--


Sent from my phone







On Thu, Feb 8, 2018 at 4:57 PM +0700, "Najih Azkalhaq" <notifications@github.com> wrote:












mas @petrabarus ? baru kepikiran...

Kalau upgrade library disini https://github.com/petrabarus/yii2-thrift-test-app

berarti harus buat release dulu ya?

Nah ini release tagnya dibuat di fork-ku atau harus di urbanindo

**Comment by azkalhaq** [comment]
owh.., pake commit hash aja gpp ya brrti?

**Comment by petrabarus** [comment]
Iya, buat test di local aja khan ya.




--


Sent from my phone







On Thu, Feb 8, 2018 at 5:07 PM +0700, "Najih Azkalhaq" <notifications@github.com> wrote:












owh.., pake commit hash aja gpp ya brrti?

**Comment by azkalhaq** [comment]
Mas @petrabarus ? kemarin buat ngecek apa sudah bisa di pakai di server sama client berdasarkan perubahan di commit ini, yang ini harus di merge dulu bukan?

**Comment by petrabarus** [comment]
Harusnya gak perlu dimerge kalo bisa set local repo. Tapi oke deh ini dimerge dulu kalo gak bisa.

**Pull request merged**